gef.py

```diff
@@ -7923,7 +7832,7 @@ def do_invoke(self, argv: List, *args, **kwargs) -> None:
             return
 
         data = [
-            ("Magic", "{!s}".format(hexdump(struct.pack(">I", elf.e_magic), show_raw=True))),
+            ("Magic", f"{hexdump(struct.pack('>I', elf.e_magic), show_raw=True)!s}"),
```

Comment:
Ah, i conflated `:s` with `!s`. In some cases where you called string on the local, if it didn't work with `:s` it should with `!s`